MyNotes/Form1.cs

```diff
@@ -68,31 +74,31 @@ private void buttonCreate_Click(object sender, EventArgs e)
         /// <param name="e"></param>
         private void buttonSave_Click(object sender, EventArgs e)
         {
-            if (File.Exists(fileDirectory.Text)) //если файл сущетсвует
```

Comment:
using "if {} else {}" usually makes code less readable.

better option would be to handle "unhappy" path and exit the method, to continue with "happy" path without extra "else" branches.

Reply:
https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88

just a general idea, not specific for c#